### DISCUSSION TITLE
Hey Vlad I found it!

### DISCUSSION THREAD

**Discussion by myndxero** [vladmandic/sdnext#1008] [General]
![image](https://github.com/vladmandic/automatic/assets/113237671/96de5129-e3ef-4a78-970a-3487feee796c)
I dunnno if you ever added it back in weeks ago after initially mentioning it, but the refresh styles button. I found it. Seems your default skin hides it. I was playing with some of the other skins out of boredom, and the fact that when I hit generate there's a 50/50 chance my comp will crash, and I noticed it's there in other skins, but not yours!

Interesting revelation I thought I'd share. I've been wicked out of the loop so maybe find it coincided with some update as I changed skins... ¯\\_(ツ)_/¯

EDIT: omg github does not like my favorite shruggy dude from discord.

EDIT 2: I had a great teacher. ¯\\\_(ツ)\_/¯

**Comment by brknsoul** [comment]
¯\\_(ツ)\_/¯

Gotta escape the underscores;

![image](https://github.com/vladmandic/automatic/assets/1969381/24507af3-0dd1-496d-91f6-90566c686594)

But yeah, you're correct;
black-orange:
![image](https://github.com/vladmandic/automatic/assets/1969381/46143e76-160a-4843-b812-7e9a002262e7)


gradio-base:
![image](https://github.com/vladmandic/automatic/assets/1969381/42799998-37f4-44bc-ad4a-7f2f8c113150)

Side request: would it be possible to change themes without restarting the server, or do too many things happen under-the-hood for that to happen?

**Reply by myndxero** [reply]
Watched that happen in real time... figure I just need to use extra \\\\\\\

**Reply by vladmandic** [reply]
lol!
ok, i've unhidden it.
re: change themes without restarting - not a chance. i wish. but gradio is what it is.
